{"name": "test_login_failure_001", "status": "passed", "start": 1744076185681, "stop": 1744076185686, "uuid": "206f9e67-e20d-4fcb-ace1-52a29736d874", "historyId": "827024c81fdee84f33955908a334e400", "testCaseId": "827024c81fdee84f33955908a334e400", "fullName": "testplatform.tests.unit.test_Login.TestLogin#test_login_failure_001", "labels": [{"name": "parentSuite", "value": "testplatform.tests.unit"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "HO"}, {"name": "thread", "value": "28800-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testplatform.tests.unit.test_Login"}]}